backend/routes/comments.js

```diff
@@ -12,17 +12,64 @@ router.use(requireAuth);
 // TODO: Is there a better way at handling the route? Currently Works as written
 router.get("/:id/comments", asyncHandler(async (req, res) => {
     const storyId = req.params.id;
-    console.log(req.params.id)
-    // console.log(storyId)
     const comments = await Comment.findAll({
         include: [
-            { model: Story, where: { id: storyId }, },
-            { model: User, attributes: ["name"] }
+            { model: Story, where: { id: storyId }, attributes: ["id", "title", "userId"] },
+            { model: User, attributes: ["id", "name"] }
         ],
         order: [["createdAt", "DESC"]],
         attributes: ["id", "body", "createdAt", "updatedAt"],
     });
     res.json({ comments });
 }));
 
+function commentNotFoundError(id) {
+    const err = Error("Comment not found");
+    err.errors = [`Comment with id of ${id} could not be found`];
+    error.title = "Comment not found";
+    err.status = 404;
+    return err;
+}
+
+router.post("/:id/comments", validateComment, asyncHandler(async (req, res) => {
+    const storyId = req.params.id;
+    const userId = req.user.id;
+    const { body } = req.body; // TODO Anything Else?
+    const comment = await Comment.create({
+        body,
+        userId,
+        storyId,
+    });
+    res.json({ comment });
+}));
+
+router.put("/:storyId/comments/:commentId", validateComment, handleValidationErrors, asyncHandler(async (req, res, next) => {
```

Comment:
the handleValidationError can be removed since it is part of the validateComment.